### DISCUSSION TITLE
Which vue-cli version comes with webpack 4?

### DISCUSSION THREAD

**Discussion by gem-mi** [vuejs/vue-cli#7173] [Q&A]
Due to some dependency missmatches I need to downgrade vue-cli to a version that has webpack 4. I managed to downgrade vue-cli from 5 to 4.5.17. Unfortunately I still seem to have webpack 5.73.0. 
Can anyone help here?

**Comment by Viggo95** [comment]
Change the dependency version of `@vue/cli-plugin` in the project's package.json to 4.5.17,  especially `@vue/cli-service`


<img width="414" alt="image" src="https://user-images.githubusercontent.com/22092110/172298407-31581f99-1e89-4ffc-9fee-604d0f992177.png">
